tests/test_PostEndpoint.py

```diff
@@ -0,0 +1,26 @@
+
+
+
+from c20_server.PostEndpoint import app
+
+
+def empty_json():
+    app.config['TESTING'] = True
+    client = app.test_client()
+
+    result = client.get('/empty_json')
+    assert result.status_code == 200
+
+def get_job_success():
+    app.config['TESTING'] = True
+    client = app.test_client()
+
```

Comment:
This is duplicated across all your tests.  It can be factored out into a `pytest` fixture.